src/procedures/ModifyPercentageExemptions.ts

```diff
@@ -9,9 +9,20 @@ import {
 } from '../types';
 import { PolymathError } from '../PolymathError';
 
+/**
+ * Procedure responsible for modifying any exception related to the Porcentage Transfer Manager module
```

Comment:
```suggestion
 * Procedure responsible for modifying any exemption related to percentage restrictions
```

Remember we are trying to abstract the module concept on the SDK. So we should mention features instead.